### PR TITLE
Switching To Debian Based Installation System (PAL Mini)

### PR DESCRIPTION
**Pull request by debashish05**
Ported PAL-USB installation to Debian from a script-based system. Debian source files are maintained at https://github.com/DreamVu/debian_installations_files/tree/3d-object-detection-tracking and https://github.com/DreamVu/Firmware/tree/I006_Tracking_JPP/deb.

### CODE REVIEW

**Review by romil611 — CHANGES_REQUESTED** [review]
The pal-mini's libPAL.so seems to have changed. Please mention the changes done here.

**Comment by debashish05** [comment]
I didn't know the new way of creating lib files, but that lib file need change.

File: code_samples/001_people_tracking.cpp [hunk 1]
```diff
@@ -40,6 +40,7 @@ using namespace std::chrono;
 
 int main( int argc, char** argv )
 {
+    printf("Update Done\n");
```
Comment (romil611) [inline code_samples/001_people_tracking.cpp:43]: Should this be here?

Comment (piyush-312) [reply code_samples/001_people_tracking.cpp:43]: just a temp update..ignore will anyway create fresh PR for both branches after testing gets completed.

**Pull request closed without merging**